docs/src/index.md

```diff
@@ -37,6 +37,6 @@ Here are some answers to some issues encountered.
 
 **A.** Scenarios are defined in a CSV file (with parameter values in columns, so that each row defines a scenario).
 
-- See the `example_scenarios.csv` file in the `examples` directory for an idea of what this looks like.
-- See `parameters.jl` file in the `examples` directory on how to extract the model specification and parameter table for a given domain.
-- See also the `running_scenarios.jl` example script which showcases how to run such a file for a given study area.
+- See the `example_scenarios.csv` file in the `test/data` directory for an idea of what this looks like.
```

Comment:
Hmm, I'm thinking we link to the file in the data-specs [repository](https://github.com/open-AIMS/ADRIA-data-specs) instead as users won't have direct access to the `test/data` directory when installed, and referring to the `test` directory is a bit awkward.

Note that the `example_scenarios.csv` file there is horribly outdated. The copy we have here is more up to date, so please submit a PR to update the example scenarios file in the data-specs repo.

Reply:
I have a push ready for the data specification repo currently don't have the permissions. 

Did you want to update just the example_scenarios.csv?

Reply:
Ah sorry, I'll give you access tomorrow morning!

Reply:
You should have access now.

And yes, just the `example_scenarios.csv` to be updated. Thank you!